### ISSUE TITLE
Different name?

### ISSUE THREAD

**Issue by msbarry** [onthegomap/flatmap#15]
The main feedback on https://news.ycombinator.com/item?id=28986762 was that "flatmap" is ambiguous and not unique. Before publishing to maven central, package managers, etc should decide whether or not to keep "flatmap" or use something different. 

Pros: 
- easier to find through search. 
- It's still early so there are too many inbound links yet

Cons:
- there are still _some_ links, forks, and stars so need to be careful to preserve those

Any suggestions?

**Comment by msbarry** [comment]
Idea: "velotile"
- https://github.com/onthegomap/velotile
- `docker run ghcr.io/onthegomap/velotile`
- maven: `com.onthegomap.velotile:velotile-core`
- package: `com.onthegomap.velotile.Main`

**Comment by msbarry** [comment]
Idea: "Global￼ Tile Engine (gte)"
- https://github.com/onthegomap/gte
- `docker run ghcr.io/onthegomap/gte`
- maven: `com.onthegomap.gte:gte-core`
- package: `com.onthegomap.gte.Main`

**Comment by danlarkin** [comment]
I agree that "flatmap" isn't a spectacular name for what this project is doing, but all my ideas are rather plain:
- mapbuilder
- vectorbuild
- tilebuilder

**Comment by msbarry** [comment]
Yeah, naming is hard :-)  It seems like a minor detail, but I am holding off on some things like publishing to maven central, and adding an OSM wiki entry until I nail down a name to stick with.

I'm leaning towards:
- something that conveys the main value proposition of the project (fast planet vector tiles)
- maybe something outdoors/running/biking related to tie it back to on the go map

Leaning away from:
- "map" in the name - saying onthegomap/flatmap out loud is a lot of "maps"
- something random unrelated to what it does (I love the mapzen projects, but it's sometimes hard to navigate things named after norse mythology)
- something with "j" or "java" in the name (I suspect most people will use this as a tool, not a library so the fact that it's written in java is inconsequential)
- something tied to how it works (like "stripe" from the stripe clipping algorithm) since that may change

**Comment by msbarry** [comment]
Probably going to go with nanotiler

https://github.com/onthegomap/nanotiler
docker run ghcr.io/onthegomap/nanotiler
maven: com.onthegomap.nanotiler:nanotiler-core
package: com.onthegomap.nanotiler.Main

**Comment by danlarkin** [comment]
What's the nano part, breaking work into small chunks?

**Comment by msbarry** [comment]
Personal significance, and (relatively) tiny resource requirements for building planet-scale vector tilesets from OpenStreetMap.

**Comment by danlarkin** [comment]
Nano makes me think small, this project is more fit for a name like Gigatiler! Or what about Planetile/Planetiler?

**Comment by msbarry** [comment]
Ooh I like `planetiler` - and I see nothing with a similar name on google search. 

```
https://github.com/onthegomap/planetiler
docker run ghcr.io/onthegomap/planetiler
maven: com.onthegomap.planetiler:planetiler-core
package: com.onthegomap.planetiler.Main
```

Just want to avoid confusion that its "planet-tiler" not "plane-tiler" (should be obvious from repo description) and if you remember the name that it's only spelled with one T.

**Comment by tomvanenckevort** [comment]
How about `globetiler`? 
Less confusing and still covers what it does.
And I can't see anything else that uses that name.

**Comment by danlarkin** [comment]
I like the portmanteau of planetiler, but yeah I guess it could also be read as plane-tiler. Though that's also sortof accurate too, right? Like it can tessellate the earth with vector tiles?

**Comment by msbarry** [comment]
Yeah I'm leaning toward planetiler, the "planet" part should be obvious from the repo description and if the plane part would be a throwback to flatmap.

**Comment by msbarry** [comment]
Planetiler it is! Updated the repo and published v0.2.0 version this morning. Thanks all for the feedback and @danlarkin for the name suggestion!
